web/daemon/server.go

```diff
@@ -252,6 +252,10 @@ func CreateServer(c *gin.Context) {
 
 	if err := programs.Create(prg); err != nil {
 		response.HandleError(c, err, http.StatusInternalServerError)
+	} else if err := prg.Scheduler.LoadMap(prg.Tasks); err != nil {
```

Comment:
Rather not use else if, but that's just me. Just chain it as if's, and do returns